fastchat/serve/gradio_web_server_vision.py

```diff
@@ -0,0 +1,982 @@
+"""
```

Comment:
In this file, can we try to import common parts from `gradio_web_server.py` to reduce the code redundancy?

Reply:
yup! i think it may be beneficial in the future to move common logic into something like `gradio_base_web_server` and then have one for `text` and one for `vision_and_language` by following more object-oriented principles, but for now I fixed this by just importing shared statements